samples/sql-persistence/transitioning-correlation-ids/SqlPersistence_2/Phase1/Shared/SharedPhase1.csproj

```diff
@@ -56,6 +56,7 @@
       <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
     </PropertyGroup>
     <Error Condition="!Exists('..\..\..\..\..\..\packages\NServiceBus.Persistence.Sql.MsBuild.2.1.2\build\NServiceBus.Persistence.Sql.MsBuild.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\..\..\..\..\packages\NServiceBus.Persistence.Sql.MsBuild.2.1.2\build\NServiceBus.Persistence.Sql.MsBuild.targets'))" />
+    <Error Condition="!Exists('..\..\..\..\..\..\packages\NServiceBus.Persistence.Sql.MsBuild.2.1.3\build\NServiceBus.Persistence.Sql.MsBuild.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\..\..\..\..\packages\NServiceBus.Persistence.Sql.MsBuild.2.1.3\build\NServiceBus.Persistence.Sql.MsBuild.targets'))" />
```

Comment:
I think the automatic script is wrong, because it doesn't remove the condition checking the old version, probably we'll need to fix this manually. But let's wait for build results.